app/src/test/java/com/example/android/psiak/DogFirebaseTest.java

```diff
@@ -0,0 +1,55 @@
+package com.example.android.psiak;
+
+import com.example.android.psiak.Model.DogFirebase;
+import com.example.android.psiak.Utils.Const;
+
+import org.junit.Test;
+
+import java.util.Hashtable;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by grzegorz.kwasniewski on 18.11.2017.
```

Comment:
Why don't we use JAVADOC instead?
`@author <a href="mailto:my_email@email.exmaple.com">Grzegorz Kwasniewski</a>`
or
`@author Grzegorz Kwasniewski`